_data/people/AdrianCrenshaw

```diff
@@ -0,0 +1,6 @@
+name: Adrian Crenshaw
+shortname: adciu
```

Comment:
Shortname and filename must match. Ideally, should match your GitHub handle (so renaming the file is likely best here)